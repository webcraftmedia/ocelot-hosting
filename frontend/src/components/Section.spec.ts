
import { renderSuspended } from '@nuxt/test-utils/runtime'
import Component from './Section.vue'
import { describe, expect, it } from 'vitest'

describe('Section', () => {
  it('renders', async () => {
    const html = await (await renderSuspended(Component)).html()
    expect(html).toMatchSnapshot()
  })
})