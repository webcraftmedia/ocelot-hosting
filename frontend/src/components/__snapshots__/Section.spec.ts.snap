// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Section > renders 1`] = `
"<div id="test-wrapper">
  <section class="mx-auto flex flex-col justify-center items-center text-5xl">
    <div class="md:container md:mx-auto"></div>
  </section>
</div>"
`;
